### PR TITLE
Update README.md to mention 'simple-obfs' installation

### PR DESCRIPTION
**Pull request by denoza**
This PR updates README.md to include the mentioning of 'simple-obfs' installation. Related to #741.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ Services Provided
   * The high-performance [libev variant](https://github.com/shadowsocks/shadowsocks-libev) is installed. This version is capable of handling thousands of simultaneous connections.
   * A QR code is generated that can be used to automatically configure the Android and iOS clients by simply taking a picture. You can tag '8.8.8.8' on that concrete wall, or you can glue the Shadowsocks instructions and some QR codes to it instead!
   * [AEAD](https://shadowsocks.org/en/spec/AEAD-Ciphers.html) support is enabled using ChaCha20 and Poly1305 for enhanced security and improved GFW evasion.
+  * [simple-obfs](https://github.com/shadowsocks/simple-obfs) plugin is installed to have robust evasion through hostile networks (esp. implementing quality of service (QOS) throttling).
```
Comment (cpu) [inline README.md]: Can I offer a few small wording tweaks?

I would start with "* **The** [simple-obfs](https://github.com/shadowsocks/simple-obfs) plugin"

"is installed to have" -> "is installed to provide"
"evasion through hostile networks" -> "traffic evasion on hostile networks"
"(esp. implementing quality of service (QOS) throttling)" -> "(especially those implementing quality of service (QoS) throttling)".

What do you think? Sorry to be picky! :-)

Comment (denoza) [inline README.md]: All fixed!

**Comment by cpu** [comment]
@alimakki Apologies for introducing another small deviation between README.md and README-fr.md - Can you update accordingly?

**Comment by alimakki** [comment]
@cpu cando.

**Pull request merged**